tests/pipeline/test_engine.py

```diff
@@ -517,17 +506,9 @@ def expected_for_col(col):
 
 
 class FrameInputTestCase(WithTradingEnvironment, ZiplineTestCase):
-    asset_ids = 1, 2, 3
-    start = Timestamp('2015-01-01', tz='utc')
-    end = Timestamp('2015-01-31', tz='utc')
-
-    @classmethod
-    def make_equity_info(cls):
-        return make_simple_equity_info(
-            cls.asset_ids,
-            start_date=cls.start,
-            end_date=cls.end,
-        )
+    asset_ids = ASSET_FINDER_EQUITY_SIDS = 1, 2, 3
+    start = ASSET_FINDER_EQUITY_START_DATE = Timestamp('2015-01-01', tz='utc')
```

Comment:
Does this test specifically want ASSET_FINDER_START_*, or could it just set the default START_DATE / END_DATE?